### DISCUSSION TITLE
Dark theme causing eye strain

### DISCUSSION THREAD

**Discussion by J-Cake** [mdn/mdn-community#297] [Browser compatibility data] [answered]
I'm lucky enough to be blessed with good eyesight, so generally I don't have an issue with taxing colour schemes or bright lights etc. but I can't help but notice that when ever I'm reading MDN with the dark theme, my eyes hurt after a few seconds, and I start to feel dizzy within a few minutes. I've never encountered this anywhere else so I assume it's related to the contrast or something technical (I have no idea about colour theory) but I thought I'd mention it. 

Perhaps lighten the blacks and dim the whites?

I do have to mention though, the light theme is 👌🏻 and the design scheme is pretty sleek.

**Comment by Josh-Cena** [comment]
Agreed. Here's a little investigation we did in Docusaurus: https://github.com/facebookincubator/infima/pull/237

**Reply by J-Cake** [reply]
That looks exactly like what I'm getting at.

**Reply by bsmth** [reply]
Interesting findings, thanks for sharing. CCing @caugner 🎨

**Reply by caugner** [reply]
With GitHub's dark mode, text has a contrast of 12.26 (13.30 in the editor), buttons have 9.86, and the headings at the top reach 15.89.

On MDN, the text has a contrast of [17.22](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=1B1B1B) (`--text-primary` on `--background-primary`), but the sidebars and the headings at the top only have [10.83](https://webaim.org/resources/contrastchecker/?fcolor=CDCDCD&bcolor=1B1B1B) (`--text-secondary` on `--background-primary`).

**Comment by caugner** [comment, marked as answer]
Thank you, I opened https://github.com/mdn/yari/issues/7705 to follow-up on this.

**Comment by pepelsbey** [comment]
It seems like the [bug has been filed](https://github.com/mdn/yari/issues/7705), and this discussion is no longer relevant. Closing as resolved.
